pkgs/development/python-modules/lingua/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, flit-core
+, polib
+, click }:
+
+buildPythonPackage rec {
+  pname = "lingua";
+  version = "4.15.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DhqUZ0HbKIpANhrQT/OP4EvwgZg0uKu4TEtTX+2bpO8=";
+  };
+
+  propagatedBuildInputs = [
+    click
+    polib
+  ];
+
+  buildInputs = [
```

```suggestion
  nativeBuildInputs = [
```
also both build- and nativeBuildInputs goes before propagatedBuildInputs